### ISSUE TITLE
Error: timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.

### ISSUE THREAD

**Issue by lessless** [webdriverio/cucumber-boilerplate#49]
Hello,

There is an error in fresh setup: 
```

 COMMAND	POST 	 "/wd/hub/session"
[14:14:32]  DATA		{"desiredCapabilities":{"javascriptEnabled":true,"locationContextEnabled":true,"handlesAlerts":true,"rotatable":true,"browserName":"chrome","loggingPrefs":{"browser":"ALL","driver":"ALL"},"requestOrigins":{"url":"http://webdriver.io","version":"4.0.9","name":"webdriverio"}}}
[14:14:33]  INFO	SET SESSION ID a5e84d1c-cc3e-44fc-8275-8dad3bc2a67e
[14:14:33]  RESULT		{"applicationCacheEnabled":false,"rotatable":false,"mobileEmulationEnabled":false,"chrome":{"chromedriverVersion":"2.21.371461 (633e689b520b25f3e264a2ede6b74ccc23cb636a)","userDataDir":"/tmp/.com.google.Chrome.A8VkqP"},"takesHeapSnapshot":true,"databaseEnabled":false,"handlesAlerts":true,"hasTouchScreen":false,"version":"50.0.2661.86","platform":"LINUX","browserConnectionEnabled":false,"nativeEvents":true,"acceptSslCerts":true,"webdriver.remote.sessionid":"a5e84d1c-cc3e-44fc-8275-8dad3bc2a67e","locationContextEnabled":true,"webStorageEnabled":true,"browserName":"chrome","takesScreenshot":true,"javascriptEnabled":true,"cssSelectorsEnabled":true}
    check default root page title
[14:14:33]  COMMAND	POST 	 "/wd/hub/session/a5e84d1c-cc3e-44fc-8275-8dad3bc2a67e/url"
[14:14:33]  DATA		{"url":"http://192.168.64.1:4000/"}
[14:14:33]  RESULT		null
      ✓ Given I open the site "/" (404ms)
[14:14:33]  COMMAND	GET 	 "/wd/hub/session/a5e84d1c-cc3e-44fc-8275-8dad3bc2a67e/title"
[14:14:33]  DATA		{}
[14:14:33]  RESULT		"Hello Ave88!"
      ✓ Then I expect that the title is "Hello Ave88!" (72ms)
[14:14:33]  COMMAND	DELETE 	 "/wd/hub/session/a5e84d1c-cc3e-44fc-8275-8dad3bc2a67e"
[14:14:33]  DATA		{}
[14:14:33]  RESULT		null
    1) "after all" hook for "Then I expect that the title is "Hello Ave88!""


  2 passing (12s)
  1 failing

  1) Test the page title "after all" hook for "Then I expect that the title is "Hello Ave88!"":
     Error: timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.
```

Selenium log: 

```
selenium_1       | 00:14:18.326 INFO - Executing: [new session: Capabilities [{rotatable=true, locationContextEnabled=true, loggingPrefs=org.openqa.selenium.logging.LoggingPreferences@4d773871, browserName=chrome, javascriptEnabled=true, handlesAlerts=true, requestOrigins={name=webdriverio, version=4.0.9, url=http://webdriver.io}}]])
selenium_1       | 00:14:18.336 INFO - Creating a new session for Capabilities [{rotatable=true, locationContextEnabled=true, loggingPrefs=org.openqa.selenium.logging.LoggingPreferences@4d773871, browserName=chrome, javascriptEnabled=true, handlesAlerts=true, requestOrigins={name=webdriverio, version=4.0.9, url=http://webdriver.io}}]
selenium_1       | Starting ChromeDriver 2.21.371461 (633e689b520b25f3e264a2ede6b74ccc23cb636a) on port 9985
selenium_1       | Only local connections are allowed.
selenium_1       | 00:14:19.376 INFO - Done: [new session: Capabilities [{rotatable=true, locationContextEnabled=true, loggingPrefs=org.openqa.selenium.logging.LoggingPreferences@4d773871, browserName=chrome, javascriptEnabled=true, handlesAlerts=true, requestOrigins={name=webdriverio, version=4.0.9, url=http://webdriver.io}}]]
selenium_1       | 00:14:19.426 INFO - Executing: [get: http://192.168.64.1:4000/])
selenium_1       | 00:14:19.786 INFO - Done: [get: http://192.168.64.1:4000/]
selenium_1       | 00:14:19.826 INFO - Executing: [get title])
selenium_1       | 00:14:19.866 INFO - Done: [get title]
selenium_1       | 00:14:19.886 INFO - Executing: [delete session: a5e84d1c-cc3e-44fc-8275-8dad3bc2a67e])
selenium_1       | 00:14:19.956 INFO - Done: [delete session: a5e84d1c-cc3e-44fc-8275-8dad3bc2a67e]

```

**Comment by rvdbogerd** [comment]
Had the same error, fixed it in PR #50

**Comment by christian-bromann** [comment]
Thanks for contributing!
